arch/arm/arm64/include/uk/asm/paging.h

```diff
@@ -104,6 +104,10 @@ struct ukarch_pagetable {
 #define PAGE_ATTR_SHAREABLE_IS		(1 << PAGE_ATTR_SHAREABLE_SHIFT)
 #define PAGE_ATTR_SHAREABLE_OS		(2 << PAGE_ATTR_SHAREABLE_SHIFT)
 
+/* Page fault error code bits */
```

Comment:
Actually, I am not so sure, I now remember that the reason why I did this is because x86 also does this in `arch/x86/x86_64/include/uk/asm/paging.h`.

Reply:
Yes, but in the scope of plat re-arch these will be moved into `arch/` too.

Reply:
I understand. IMO this should still stay this way to be consistent with x86. When plat re-arch comes, I believe this migration should be all part of the same commit. Therefore, I would like to keep it like this for now, because there is going to be some time until plat re-arch comes around.